bin/clover/src/pipeline-steps/generateAssetFuncs.ts

```diff
@@ -52,11 +52,6 @@ export function generateAssetFuncs(
 function generateAssetCodeFromVariantSpec(
   variant: ExpandedSchemaVariantSpec,
 ): string {
-  if (variant.domain.kind !== "object") throw "Domain prop is not object";
```

Comment:
We don't need these checks anymore because domain and resourceValue are specified as objects in the type system.